test/unittests/CMakeLists.txt

```diff
@@ -228,6 +228,7 @@ include_directories("${PROJECT_SOURCE_DIR}")
 include_directories("${PROJECT_SOURCE_DIR}/src/lib")
 include_directories("${PROJECT_SOURCE_DIR}/src/app")
 include_directories("${PROJECT_SOURCE_DIR}/test/unittests")
+include_directories("${PROJECT_SOURCE_DIR}/test")
```

Comment:
This is for the rule about including the directory in the #include directive.
```
#include "unittest.h"
```
Must be:
```
#include "unittest/unittest.h"
```
And for this, we need the proper `-I`, which this modification gives us.